### DISCUSSION TITLE
Gutenberg contributions

### DISCUSSION THREAD

**Discussion by erikyo** [wp-blocks/wp-blocks#6] [General]
Seems that the Gutenberg project is open to contributions, so it is possible that in the future we may propose something of our own 

https://github.com/WordPress/gutenberg/issues/39915#issuecomment-1283793782

**Comment by johnhooks** [comment]
It is a goal of mine to be come a contributor to Gutenberg. So yes, exactly, if we create something we think is special, I would like to try to merge it into Gutenberg if there is a desire to do so in the community. But also I want to build it outside the project, so we can move fast and make our own decisions.

**Reply by erikyo** [reply]
Perfect! 100% reflects the way in which I see the project, outside Gutenberg to have our freedoms but with the aim of contributing with the best repos to the core.
